### DISCUSSION TITLE
Watchdog init compilation error

### DISCUSSION THREAD

**Discussion by Burdi60** [espressif/arduino-esp32#9818] [General]
In the latest versions of the ESP32 card manager (version 3.0.1 and 3.0.2), compiling the esp_task_wdt_init() function generates an error.
There are no compilation errors with earlier versions (2.0.17 for example).
Has anyone come across this problem before ?

**Comment by f5exo** [comment]
Yes, I confirm, since 3.0.0 and 3.0.1 For M2Core2 the simple display example causes a compilation error.
2.0.17 is good.

**Reply by Burdi60** [reply]
Thank you for this confirmation.
Has anyone found a workaround?
